engine/lib/luajit-ffi-gen/src/impl_item/generate_rust.rs

```diff
@@ -80,7 +80,7 @@ impl ImplInfo {
         quote! {
             #[no_mangle]
             pub unsafe extern "C" fn #func_ident(#self_token #(#param_tokens),*) #ret_token {
-                tracing::trace!("Calling: {}", #func_name);
+                // tracing::trace!("Calling: {}", #func_name);
```

Comment:
Do you still need it?

Reply:
Oh, I didn't mean to comment this out. I think it's still useful to have.